src/picker.js

```diff
@@ -13,7 +13,8 @@ angular.module("ion-datetime-picker", ["ionic"])
                 hourStep: "=?",
                 minuteStep: "=?",
                 secondStep: "=?",
-                onlyValid: "=?"
+                onlyValid: "=?",
+                telTypeForTimeFields: "=?"
```

Comment:
Remove this line (because all boolean attrs are handled differently, see below).